lib/constants/constants.dart

```diff
@@ -1,34 +1,52 @@
+import 'dart:ui';
+
 class Constants {
-  // Poster and Backdrop
-  static const posterLocation ="assets/images/poster.jpg";
-  static const backdropLocation = "assets/images/back.jpg";
-  static const posterYAligment = 20.0;
-  static const posterXAligment = 15.0;
-  static const posterHeight = 190.0;
-  static const posterWidth = 140.0;
-  static const posterStroke = 1.0;
+  // App colors
+  static const Color c1 = Color(0xFF001D3D);
+  static const Color c2 = Color(0xFF003566);
```

Comment:
Choose more suitable names for this constants

Reply:
Fixed. I will implement default colors instead.